.github/workflows/tests.yml

```diff
@@ -82,3 +82,18 @@ jobs:
           flags: ${{ matrix.python-version }}
           token: ${{ secrets.CODECOV_TOKEN }}
           verbose: true
+  notifications:
+    name: Regression notifications
+    runs-on: ubuntu-latest
+    needs: build
+    if: failure() && github.ref == 'refs/heads/zimeg-ci-notifications' && github.event_name != 'workflow_dispatch'
```

Comment:
📣 `workflow_dispatch` will not send a notification!

🔗 https://github.com/slackapi/bolt-python/actions/runs/15935708998